pkg/client/helper.go

```diff
@@ -240,13 +240,18 @@ func InClusterConfig() (*Config, error) {
 	if err != nil {
 		return nil, err
 	}
+	rootCA, err := ioutil.ReadFile("/var/run/secrets/kubernetes.io/serviceaccount/ca.cert")
+	if err != nil {
+		return nil, err
+	}
 	return &Config{
 		// TODO: switch to using cluster DNS.
 		Host:        "https://" + net.JoinHostPort(os.Getenv("KUBERNETES_SERVICE_HOST"), os.Getenv("KUBERNETES_SERVICE_PORT")),
```

Comment:
It contains quite a few names including the service ip https://github.com/GoogleCloudPlatform/kubernetes/pull/10107